### DISCUSSION TITLE
backup time config-sample

### DISCUSSION THREAD

**Discussion by GuillemFemenias** [TriliumNext/Trilium#4014] [Q&A] [answered]
Hi, is there any way to set at what exact time i want the backups to be created? I guess its editing something in the config-sample but i didn't find any info, thank you!

**Comment by Nriver** [comment, marked as answer]
It's not configurable.

**Comment by zadam** [comment]
Hello, @Nriver is correct.

However, I just added a possibility to create a backup via [ETAPI](https://github.com/zadam/trilium/wiki/ETAPI) call which will be released in the next 0.60 version. This should allow you to automate the process via e.g. cron exactly as you wish.

**Reply by GuillemFemenias** [reply]
Awesome, thank you!
